db/migrate/20150910153517_add_full_name_to_custom_attributes.rb

```diff
@@ -0,0 +1,5 @@
+class AddFullNameToCustomAttributes < ActiveRecord::Migration
+  def change
+    add_column :custom_attributes, :full_name, :text
```

Comment:
How about `:unique_name` instead of `:full_name`?

To me `full_name` implies a person's full name.  And, the real reason for this attribute is to tell one custom attribute apart from another custom attribute with the same `section` and `name`.